.github/workflows/run-audit.yaml

```diff
@@ -0,0 +1,29 @@
+name: Security Audit
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  audit:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install jq
+        run: sudo apt-get update && sudo apt-get install jq
+
+      - name: Run security audit
+        run: |
+          yarn install --frozen-lockfile
+          yarn audit --json > audit-report.txt
+        continue-on-error: true
+
+      - name: Validate audit report
```

Comment:
Can we add a comment to describe why separate script for auditon is needed?

Reply:
Done